BookingsApi/BookingsApi.UnitTests/Controllers/JudiciaryPersonControllerTests.cs

```diff
@@ -0,0 +1,278 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Controllers;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Controllers
+{
+    public class JudiciaryPersonControllerTests
+    {
+        protected JudiciaryPersonController Controller;
```

Comment:
This class isn't abstract, any reason why the properties are protected?

Reply:
done